client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -819,6 +819,7 @@ const mapStateToProps = (
 		answer_prompt: getQueryArg( window.location.href, 'answer_prompt' ),
 		completedFlow,
 		canvas: getQueryArg( window.location.href, 'canvas' ), // Site editor can initially load with or without nav sidebar (Gutenberg v15.0.0)
+		showLaunchpad: getQueryArg( window.location.search, 'showLaunchpad' ),
```

Comment:
This redirects the calypso URL parameter `showLaunchpad` to the iframe URL so we can get it on the `iframe-bridge-server.js`. 

It will be outdated as soon as we remove the iframe on the editor. Context: pb5gDS-2NW-p2#comment-3638

Reply:
Wondering if we should add a comment to the code here for context in case the iframe was removed suddenly.

Reply:
Good point @candy02058912. As we decide to move it without the iFrame, I just removed that code. Thanks!